dialog/file.go

```diff
@@ -65,7 +66,7 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 			info, err := os.Stat(path)
 			if os.IsNotExist(err) {
 				f.win.Hide()
-				callback(fyne.SaveFileToURI("file://" + path))
+				callback(storage.SaveFileToURI(storage.NewURI("file://" + path)))
```

Comment:
it might be worth adding storage.NewFileURI(path)

Reply:
Maybe, can we leave it out until we see if anyone needs that? It would only be useful externally if people are constructinhg URIs to feed in - which is not a primary path ATM.